### ISSUE TITLE
SSH: wrong ~/.ssh folder mapping in the container?

### ISSUE THREAD

**Issue by ebaschiera** [home-assistant/hassio-addons#1034]
Hi,
I noticed that in SSH Server addon, there is a persistent .ssh in /data.
Il looks like in the container that /data/.ssh folder is mapped to /root/.ssh/.ssh , while in my opinion it should be mapped to /root/.ssh.

This is what I can see in terminal:

```
~ $ pwd
/root
~ $ ls -all .ssh/
total 12
drwxr-xr-x    2 root     root          4096 Feb  1 19:31 .
drwx------    1 root     root          4096 Feb  1 19:31 ..
lrwxrwxrwx    1 root     root            10 Feb  1 19:31 .ssh -> /data/.ssh
-rw-------    1 root     root           396 Feb  1 19:30 authorized_keys
```

I took a look at https://github.com/home-assistant/hassio-addons/blob/master/ssh/data/run.sh#L81 but I couldn't spot the issue... 

I am not sure this is a bug or it's an intended behaviour... I'd say it's not intended.

Thanks!

**Comment by frenck** [comment]
I cannot reproduce this on my end, tested on multiple systems.

![image](https://user-images.githubusercontent.com/195327/73651939-dbc5f600-4685-11ea-9f2a-514440b2cedc.png)

**Comment by pvizeli** [comment]
Fix with 8.2
